src/RdfSkosFieldHandler.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\rdf_skos;
+
+use Drupal\rdf_entity\Exception\NonExistingFieldPropertyException;
+use Drupal\rdf_entity\RdfFieldHandler;
+use Drupal\rdf_entity\RdfFieldHandlerInterface;
+
+/**
+ * RDF field handler for SKOS entities.
+ */
+class RdfSkosFieldHandler extends RdfFieldHandler {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function buildEntityTypeProperties($entity_type_id): void {
+    if (!empty($this->outboundMap[$entity_type_id]) && !empty($this->inboundMap[$entity_type_id])) {
+      return;
+    }
+
+    $mapping = $this->getSkosPredicateMappings($entity_type_id);
+    $this->outboundMap[$entity_type_id] = $this->inboundMap[$entity_type_id] = [];
+    // We double up the entity type ID instead of the bundle.
+    $this->outboundMap[$entity_type_id]['bundles'][$entity_type_id] = $mapping['rdf_type'];
+    // We don't have bundles but a key is expected.
+    $this->outboundMap[$entity_type_id]['bundle_key'] = NULL;
+    $this->inboundMap[$entity_type_id]['bundles'][$mapping['rdf_type']][] = $entity_type_id;
+    $field_definitions = $this->entityFieldManager->getBaseFieldDefinitions($entity_type_id);
+
+    foreach ($field_definitions as $field_name => $definition) {
+      $field_mapping = isset($mapping['fields'][$field_name]) ? $mapping['fields'][$field_name] : NULL;
+      if (!$field_mapping) {
```

Comment:
`if (NULL === $field_mapping)`

Reply:
Nothing wrong with the way it is.

Reply:
It's linked to my other comment, I think we should make the conditions more verbose... using the exclamation mark can be really weird sometimes, see: https://3v4l.org/rTLlU